src/components/grids.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const gapSize = 20
+const numberOfColumns = 12
+
+export const Row = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-right: -${gapSize / 2}px;
+  margin-left: -${gapSize / 2}px;
+  margin-bottom: ${gapSize}px;
+`
+
+export const Col = styled.div`
+  box-sizing: border-box;
+  flex: 0 0 auto;
+  padding-right: ${gapSize / 2}px;
+  padding-left: ${gapSize / 2}px;
+  flex-basis: ${props => (100 / numberOfColumns) * (props.size)}%;
+  max-width: ${props => (100 / numberOfColumns) * (props.size)}%;
```

Comment:
Rather than doing math here and having the user of the API having to define `props.size`, I wonder if we could just use `flex-grow` here? https://css-tricks.com/snippets/css/a-guide-to-flexbox/#article-header-id-10

The size prop can still exist, it would just default to 1 so the user doesn’t need to specify it if they want equal distribution.